views/default/forms/admin/menu/save.php

```diff
@@ -7,11 +7,7 @@
 $num_featured_items = 6;
 
 // get site menu items
-$menu = elgg_get_config('menus');
-$menu = $menu['site'];
-$builder = new ElggMenuBuilder($menu);
-$menu = $builder->getMenu('name');
-$menu_items = $menu['default'];
+$menu_items = elgg_get_menu('site')->getMenuItems('name', 'default');
```

Comment:
i even think it is better and may even be considered a bug fix. The intention is to get the registered site menu items... but before the hook was not triggered, so some menu items could be missing